tests/unit/src/main/scala/tests/TestingClient.scala

```diff
@@ -244,6 +260,9 @@ final class TestingClient(workspace: AbsolutePath, buffers: Buffers)
           CheckDoctor.moreInformation
         } else if (SelectBspServer.isSelectBspServer(params)) {
           params.getActions.asScala.find(_.getTitle == "Bob").get
+        } else if (isSameMessageFromList(ChooseBuildTool.params)) {
+          params.getActions.asScala.toList.headOption
```

Comment:
Could we look for `sbt` instead? If the order changes at any point the test will fail.

Reply:
Ah, good point 👍